contrib/pyln-testing/pyln/testing/fixtures.py

```diff
@@ -432,36 +432,28 @@ def is_msat_or_any_or_currency(checker, instance):
                                         type_checker=type_checker)
 
 
-def _load_schema(filename, is_request):
+def _load_schema(filename):
     """Load the schema from @filename and create a validator for it"""
     with open(filename, 'r') as f:
-        return _extra_validator(is_request)(json.load(f))
+        data = json.load(f)
+        return [_extra_validator(True)(data.get('request', {})), _extra_validator(False)(data.get('response', {}))]
 
 
 @pytest.fixture(autouse=True)
 def jsonschemas():
-    """Load schema files if they exist: returns request/response schemas by pairs"""
+    """Load schema file if it exist: returns request/response schemas by pairs"""
     try:
         schemafiles = os.listdir('doc/schemas')
     except FileNotFoundError:
         schemafiles = []
 
     schemas = {}
     for fname in schemafiles:
-        if fname.endswith('.schema.json'):
-            base = fname.rpartition('.schema')[0]
-            is_request = False
-            index = 1
-        elif fname.endswith('.request.json'):
-            base = fname.rpartition('.request')[0]
-            is_request = True
-            index = 0
-        else:
-            continue
-        if base not in schemas:
+        if fname.startswith('lightning-') and fname.endswith('.json'):
+            base = fname.replace('lightning-', '').replace('.json', '')
+            # Request is 0 and Response is 1
             schemas[base] = [None, None]
```

Comment:
This line can also be deleted?

Reply:
These lines should not be deleted to ensure that the `base` is getting the correct rpc method name and only rpc files in the schema are being processed.

Reply:
I mean, the code now looks like:

```
            schemas[base] = [None, None]
            schemas[base] = _load_schema(os.path.join('doc/schemas', fname))
```

The first line of these can be deleted!

Reply:
Got it :). Removed `schemas[base] = [None, None]`.